### PR TITLE
Add custom widgets for data visualization

### PR DESCRIPTION
**Pull request by felixp8**

### CODE REVIEW

**Comment by CodyCBakerPhD** [comment]
@felixp8 Would you mind putting together a video recording that demo's these custom widgets? It makes it super easy and reliable to reference them in the future (also feel free to embed it in the README for easy access)

**Comment by felixp8** [comment]
sure I'll try to do that!

**Comment by CodyCBakerPhD** [comment]
@felixp8 In the new video, is there a way to make the default width of those table columns large enough to naturally show the column names without needing to be expanded? Does it do that automatically if you expand your screen size or zoom?

**Comment by felixp8** [comment]
fortunately `ipydatagrid` has a kwarg for rescaling column widths for column name, added now

**Comment by felixp8** [comment]
misclick...

File: README.md [hunk 1]
```diff
@@ -102,3 +102,28 @@ User configuration of the conversions primarily involves editing/creating YAML f
...
+
+#### Trial alignment for processed electrophysiology data
+
+While `nwbwidgets` supports trial alignment and averaging for spiking data, it does not for generic TimeSeries, so the `AlignedAveragedTimeSeriesWidget` offers this functionality, likely most useful for the various processed electrophysiology data computed during the experiment.
+
+https://github.com/catalystneuro/stavisky-lab-to-nwb/assets/64850082/75aa160d-c054-4acd-a99e-2424c5e41792
+
+#### Decoder RNN and language model predictions over time
+
+The `DecodingOutputWidget` displays the phoneme probabilities and the words predicted by the decoder RNN and language model at each step.
```
Comment (CodyCBakerPhD) [inline README.md]: For this widget, can you add something clarifying what the strikethrough's mean on the displayed words on the top interval?

**Comment by CodyCBakerPhD** [comment]
These look amazing! The README is super convenient for referencing now

One minor comment for describing the last video and this should be good to go

File: README.md (hunk 1 above)
Comment (felixp8) [reply README.md]: yup will do

**Comment by felixp8** [comment]
clearer now hopefully

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**